firefox-ios/Client/PrivacyWindowHelper.swift

```diff
@@ -0,0 +1,25 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+final class PrivacyWindowHelper {
+    private var privacyWindow: UIWindow?
+
+    func showWindow(withThemedColor color: UIColor) {
+        guard let windowScene = UIWindow.attachedKeyWindow?.windowScene else { return }
+
+        privacyWindow = UIWindow(windowScene: windowScene)
+        privacyWindow?.rootViewController = UIViewController()
+        privacyWindow?.rootViewController?.view.backgroundColor = color
+        // Set the privacy window level to be above alert windows (highest in importance).
+        privacyWindow?.windowLevel = .alert + 1
+        privacyWindow?.makeKeyAndVisible()
```

Comment:
@PARAIPAN9 Did we investigate any alternatives for coercing iOS into refreshing the thumbnails? I know we had a similar bug to this a while back for `SensitiveViewController`.

I'm seeing a couple oddities with this fix in place and using stage manager on iPad. See video below: the thumbnail for Incognito is sometimes blank but other times is showing the content, and then when switching to the app switcher in some cases the normal browsing window is actually blanked out in addition to the private one (happens at 0:15).

https://github.com/user-attachments/assets/bb34ae81-7761-46b7-8758-5222da92f1f2

Reply:
Thanks @mattreaganmozilla, didn't think of this scenarios.

Reply:
@PARAIPAN9 FYI based on the analytics data we have right now I don't believe many users are actually using multi-window on iPad. Probably would want to double-check with Andy but I don't think any of these concerns are a blocker for the fix.